### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by kaa7**

### CODE REVIEW

File: extended.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Fizz Buzz++
```
Comment (cipacda) [inline extended.js:1]: We should not commit compiled files (.js in this case) as this is redundant and may be different from machine to machine (i.e. on UNIX will have different line endings)

File: rules.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        acc.push("Bang")
+    }
+}
+
+class RuleBong implements Rule {
+    match(x: number): boolean {
+        return x % 11 == 0;
+    }
+
+    apply(acc: string[]): void {
```
Comment (cipacda) [inline rules.ts:53]: We could come up with a better name for `acc`

**Review by cipacda — APPROVED** [review]
Nicely done, I like the classes version a lot

File: rules.ts (hunk 2 above)
Comment (kaa7) [reply rules.ts:53]: Acc comes from accumulator, I will try to think of a better name (maybe cur_res?).

**Pull request merged**